src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/validation/YFlowValidator.java

```diff
@@ -153,6 +154,20 @@ private void checkBandwidth(YFlowRequest yFlowRequest)
         }
     }
 
+    private void checkMaxLatency(YFlowRequest yFlowRequest) throws InvalidFlowException {
+        if (yFlowRequest.getMaxLatency() == null || yFlowRequest.getMaxLatencyTier2() == null) {
+            return;
+        }
```

Comment:
I think also we should throw exception if maxLatency is null and maxLatencyTier2 is not null

Reply:
done